internal_transfers/actions/tests/database.spec.ts

```diff
@@ -43,7 +44,8 @@ describe("test database insertion methods", () => {
       { solver: solver, logIndex: 0 }
     );
 
-    const results = await db.query(sql`SELECT * from settlements;`);
+    const results = await db.query(sql`SELECT *
+                                           from settlements;`);
```

Comment:
Seems like a regression to me.

Reply:
This was the linter I think... but I will try to put it back.